bedrock/externalpages/urls.py

```diff
@@ -22,4 +22,5 @@
     page("pocket/privacy/", "externalpages/pocket/privacy.html"),
     page("pocket/tos/", "externalpages/pocket/tos.html"),
     page("pocket/save-to-pocket/", "externalpages/pocket/save-to-pocket.html"),
+    page("analytics/", "externalpages/analytics/index.html"),
```

Comment:
Nit: I know it's no longer as easy to add test-only views to Django, but if the path here is only used for testing, it might worth making it explicit, for our grandkids' sake 😉 

```suggestion
    page("analytics/", "externalpages/analytics-tests/index.html"),
```

Reply:
@nathan-barrett To avoid a clash with my work on Pocket Mode, where we'll be losing the idea of an app called 'externalpages', it'd be better tomove this into mozorg's URLconf, as mentioned [here](https://github.com/mozilla/bedrock/pull/11494#discussion_r850544693)